src/Microsoft.DotNet.XHarness.CLI/CommandArguments/Android/AndroidInstallCommandArguments.cs

```diff
@@ -34,6 +43,18 @@ protected override OptionSet GetTestCommandOptions() => new()
                 "device-id=", "Device where APK should be installed",
                 v => DeviceId = v
             },
+            {
+                "boot-timeout=", "Timeout in seconds to wait for a device after boot competion",
```

Comment:
In iOS we call this `launch-timeout`, if it makes sense for your flow, we should unify naming

Reply:
done